NOMIS/nomis.go

```diff
@@ -301,3 +307,25 @@ func downloadFile() {
 
 	fmt.Printf("Downloaded a file %s with size %d", fileName, size)
 }
+
+func checkSubString (res CenStructure, index0, indx int)  string {
+	var strArray []string
+	var result string
+	value :=strings.Contains(res.Structure.Keyfamilies.Keyfamily[index0].Annotations.Annotation[indx].Text.(string), "http")
+	if value{
+		str1:=strings.Split(res.Structure.Keyfamilies.Keyfamily[index0].Annotations.Annotation[indx].Text.(string), "http")
```

Comment:
formatting

Reply:
General observation You're not using `value` after you've performed the `if` check. You could do this in one statement and save some code:
```
if strings.Contains(res.Structure.Keyfamilies.Keyfamily[index0].Annotations.Annotation[indx].Text.(string), "http") {
...
}
```
Also `value` is not a descriptive var name something along the lines of `containsVal`  feels it more accurately describes the purpose of the var. I'm sorry if this feels like a nit picky point but I place a lot value in well named variables.